python/cudf/cudf/core/frame.py

```diff
@@ -3199,9 +3199,9 @@ def ceil(self):
         """
 
         warnings.warn(
-            "Series.ceil and DataFrame.ceil are deprecated and will be \
-                removed in the future",
-            DeprecationWarning,
+            "Series.ceil and DataFrame.ceil are deprecated and will be "
+            "removed in the future",
+            FutureWarning,
```

Comment:
Note to reviewers: We standardized on `FutureWarning` (appears at runtime) instead of `DeprecationWarning` (silent by default) in #9392.